mitol-django-common/mitol/common/models.py

```diff
@@ -37,7 +37,7 @@ class Meta:
         abstract = True
 
 
-def _items_for_class(content_type_field, items, model_cls):
+def _items_for_class(content_type_field: str, items, model_cls):
```

Comment:
Add type hints for the other args and return type?

Reply:
I adjusted some of the other typing in this file based on related code in the `django-stubs` type library.